types/twine-sugarcube/test/events.ts

```diff
@@ -0,0 +1,18 @@
+$(document).on(":passageinit", event => {
+	event.passage; // $ExpectType Passage
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/test/events.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.